storage/src/mmr/proof.rs

```diff
@@ -286,15 +290,15 @@ impl<D: Digest> Proof<D> {
         &self,
         hasher: &mut H,
         elements: &[E],
-        start_loc: u64,
+        start_loc: Location,
         root: &I::Digest,
-    ) -> Result<Vec<(u64, D)>, super::Error>
+    ) -> Result<Vec<(Position, D)>, super::Error>
     where
         I: CHasher<Digest = D>,
         H: Hasher<I>,
         E: AsRef<[u8]>,
     {
-        let mut collected_digests = Vec::new();
+        let mut collected_digests: Vec<(Position, D)> = Vec::new();
```

Comment:
does type inference no longer work here?

Reply:
Removed the type annotation